masterlist.yaml

```
@@ -4063,6 +4063,43 @@ plugins:
         subs: [ 'Remove **00032650**, **00069AEE**, & **00069AEF** from **Variant Vagabonds.esp**.' ]
         condition: 'checksum("Variant Vagabonds.esp", E5228022)'
 
+  - name: 'VASE - .*\.es(m|p)'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/28186/'
+        name: 'VAcuitys Spirits and Ephemerals'
```

It looks like this mod has actually been incorporated into MMM at some point, & the readme for the unofficial MMM patch states:

> I'm sure Corepc will come and fix you up with an official MMM patch sooner or later.  Then when these new creatures get integrated into MMM there won't *be* any patch file needed any longer.

It's therefore preferable to redirect users of both to the official MMM add-on. FCOM probably complicates things, so that'll have to be properly handled at a later time.

```suggestion
        name: 'VAcuitys Spirits and Ephemerals'
    msg:
      - <<: *alreadyInMMM
        condition: 'active("Mart''s Monster Mod.esm") and not active("FCOM_Convergence.esm") and file("Mart''s Monster Mod - Divserse Wisp and VASE Addon.esp")'
      - <<: *useInstead
        subs: [ '[**Mart''s Monster Mod - Divserse Wisp and VASE Addon.esp**](https://www.nexusmods.com/oblivion/mods/17784/)' ]
        condition: 'active("Mart''s Monster Mod.esm") and not active("FCOM_Convergence.esm") and not file("Mart''s Monster Mod - Divserse Wisp and VASE Addon.esp")'
```